lib/mdot/client.rb

```diff
@@ -51,16 +52,25 @@ def get_supplies
     #
     def submit_order(request_body)
       with_monitoring_and_error_handling do
-        perform(:post, @supplies, request_body, headers).body
+        perform(:post, @supplies, request_body, submission_headers).body
```

Comment:
Just to verify, this POST request only needs the VA_API_KEY token and no longer needs the headers referenced on line 61, correct?

Reply:
yes, that is correct.